### PR TITLE
Wire no form page to feature toggle

### PR DESCRIPTION
**Pull request by ToddWebDev**
# THIS IS A NEW FORM NOT IN PRODUCTION

## Summary
Add NoFormPage to wire up feature toggle on Income and Asset Statement Form (21P-0969)

## Issue
[Generate new vets-website application#81941](https://github.com/department-of-veterans-affairs/va.gov-team/issues/81941)

### Related Issues
[Add feature toggle #81935](https://github.com/department-of-veterans-affairs/va.gov-team/issues/81935)

### Epic
[Digitize Income and Asset Statement Form 21P-0969](https://app.zenhub.com/workspaces/benefits-pension-64e775aaa6b1ca1ed49b2ede/issues/gh/department-of-veterans-affairs/va.gov-team/80925)

### Screenshots
| Introduction Page | No Form Page |
| ---------- | --------- |
|localhost_3001_income-and-asset-statement-form-21p-0969_introduction (1)|localhost_3001_income-and-asset-statement-form-21p-0969_introduction|

## How to run in local environment
1. Check out this branch locally in 'vets-website'
2.  Checkout this [branch](https://github.com/department-of-veterans-affairs/content-build/pull/2075) locally in 'content-build'
3. Run 'vets-api' in your local environment
4. Run the 'vets-website' app with `yarn watch --env entry=static-pages,auth,login-page,profile,dashboard,income-and-asset-statement`

## How to toggle feature flag in local environment
1. Go to [income_and_assets_form_enabled](http://localhost:3000/flipper/features/income_and_assets_form_enabled)

## How to verify
1. Go to `http://localhost:3001/income-and-asset-statement-form-21p-0969` in your browser
2. Verify the 'Introduction' page is loaded when income_and_assets_form_enabled is enabled
3. Verify the 'No Form' page is loaded when income_and_assets_form_enabled is disabled

## Testing
- [x]  Unit tests passing

## What areas of the site does it impact?
Income and Asset Statement Application

### Quality Assurance & Testing
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling
- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication
- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Review by TaiWilkin — CHANGES_REQUESTED** [review]
This is working well. One small change request for some testing syntax.

File: src/applications/income-and-asset-statement/tests/NoFormPage.unit.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  it('should render', () => {
+    const store = mockStore({
+      user: { login: { currentlyLoggedIn: true } },
+    });
+    const { container } = render(
+      <Provider store={store}>
+        <NoFormPage />
+      </Provider>,
+    );
+    waitFor(() => {
```
[the remark this answers is not in the document]
Comment (ToddWebDev) [reply src/applications/income-and-asset-statement/tests/NoFormPage.unit.spec.jsx:22]: Good catch! Thank you.

**Review by TaiWilkin — APPROVED** [review]

**Comment by ataker** [comment]
Waiting for checks to pass before reviewing.

**Comment by ToddWebDev** [comment]
@ataker Relevant checks are passing now. The unit test checks will pass after future work to add unit tests. This is a generated app with scaffolding only.

**Review by ataker — APPROVED** [review]

**Pull request merged**